pkg/controller/build/osbuildcontroller_test.go

```diff
@@ -246,86 +246,6 @@ func TestOSBuildControllerFailure(t *testing.T) {
 	})
 }
 
-// This test checks that a previously built MachineOSBuild can be reused
-// without performing another build provided that the hashed name is the same.
-func TestOSBuildControllerReusesPreviouslyBuiltImage(t *testing.T) {
```

Comment:
@cheesesashimi I had to delete this test as I couldn't figure out how to replicate an actual registry where skopeo inspect would work. There is an e2e test for this scenario so makes sense to rely on that to test this functionality.